src/UI/Art/SpriteArtist/SpriteUIUtils.java

```diff
@@ -108,6 +117,51 @@ public static BufferedImage makeTextMenu(Color bg, Color frame, Color focus, Arr
     return menuImage;
   }
 
+  /**
+   * Returns an image with the input string printed within the specified width, in normal text.
+   * @param reqWidth: Actual UI size in pixels that you want to fit the text into.
+   */
+  public static BufferedImage paintTextNormalized(String prose, int reqWidth)
+  {
+    // Figure out how big our text is.
+    int drawScale = SpriteOptions.getDrawScale();
+    int characterWidth = SpriteLibrary.getLettersUppercase().getFrame(0).getWidth() * drawScale;
+    int characterHeight = SpriteLibrary.getLettersUppercase().getFrame(0).getHeight() * drawScale;
+
+    ArrayList<String> lines = new ArrayList<String>();
+    // Unload our prose into the lines it already has
+    lines.addAll(Arrays.asList(prose.split("\\R"))); // \R matches all newline formats, yay convenience
```

Comment:
Java 7 doesn't support `"\R"`.  How about `"[\n\r]+"`?

Reply:
Hmm. It works just fine on my end.

Reply:
Are you sure you are running 7? My laptop chokes on it.

Reply:
Looks like it to me, boss.
![image](https://user-images.githubusercontent.com/8674241/57947829-9a967780-78a5-11e9-803e-93649e0c8d91.png)

Reply:
Huh. That matches what mine says. And yet...
```
Exception in thread "main" java.util.regex.PatternSyntaxException: Illegal/unsupported escape sequence near index 1
\R
 ^
	at java.util.regex.Pattern.error(Unknown Source)
	at java.util.regex.Pattern.escape(Unknown Source)
	at java.util.regex.Pattern.atom(Unknown Source)
	at java.util.regex.Pattern.sequence(Unknown Source)
	at java.util.regex.Pattern.expr(Unknown Source)
	at java.util.regex.Pattern.compile(Unknown Source)
	at java.util.regex.Pattern.<init>(Unknown Source)
	at java.util.regex.Pattern.compile(Unknown Source)
	at java.lang.String.split(Unknown Source)
	at java.lang.String.split(Unknown Source)
```
Weird.

Reply:
It... could be that I'm using the Zulu stuff rather than the Oracle stuff?
My reason for switching was their change in license I read about recently.

Reply:
Huh. Never heard of Zulu stuff. They are just a... repackaged OpenJDK?

Also seems odd that they named themselves after a time zone.

Reply:
Yes, but with different licensing.
If you want me to change this, I guess I can, but it looks like it's Java 7 compliant to me, hah.

Reply:
Also, matching `"[\n\r]+"` would negate the ability to add multiple-line breaks.

Reply:
`"\r?\n?"` would technically match nothing, wouldn't it? If you want to match against just one of a given character set, just drop the '+': `"[\n\r]"`

Reply:
But then `\r\n`, the Windows newline, will produce 2 lines.
Based on [this question](https://stackoverflow.com/questions/3445326/regex-in-java-how-to-deal-with-newline), I'm thinking `"\r\n|\n|\r"`

Reply:
Works for me.